_drafts/autolab-and-docker.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Autolab and Docker"
+author: Mihir Pandya
+
+tags: [autograding, docker, tango]
+---
+
+![]({{site.baseurl}}/assets/docker.svg)
+
+As highlighted in our ['Hello World' post][hello-world], we at Autolab want to make Autograding for All a reality. In pursuit of this vision, we started by making Autolab robust enough for a variety of classes. Following that, we [open-sourced][gh-autolab] this project after it had been tried and tested at Carnegie Mellon University. 
```

Comment:
I actually think this is the perfect length introduction, and a pretty powerful introduction.